selenium_tests/cross_listing/page_objects/cross_listing_index_page.py

```diff
@@ -0,0 +1,12 @@
+from selenium.webdriver.common.by import By
+
+from selenium_tests.cross_listing.page_objects.cross_listing_base_page_object\
+    import CrossListingBasePageObject
+
+
+class Locators(object):
+    HEADING_ELEMENT = (By.XPATH, '//h3[contains(.,"Cross Listing")]')
+
+
+class IndexPagePageObject(CrossListingBasePageObject):
```

Comment:
It's supposed to model the landing (home) page of the cross-listing tool.  I've modified the page object name to "main_page_object" and added a quick comment on what it's suppose to model.   There is only ONE page to the cross-listing tool right now (so the index page seems intuitive to me...)   The "heading_element" is there is to cover the _is_loaded condition (to check test is in the right place).  I think this PO will be much clearer once more tests are in place.